### PR TITLE
Added a Vorkath Plugin

### PR DESCRIPTION
**Pull request by Justin-Schneider**
- Shows how many attacks vorkath has until his next phase
- Shows what his next phase is once it can be determined
- Highlights the Zombified Spawn with color and the crumble undead icon
- Screenshots below

Feedback is welcome. 

screenshot 30
screenshot 29
screenshot 28
screenshot 27

### CODE REVIEW

File: runelite-api/src/main/java/net/runelite/api/Skill.java [hunk 1]
```diff
@@ -53,6 +53,9 @@
 	RUNECRAFT("Runecraft"),
 	HUNTER("Hunter"),
 	CONSTRUCTION("Construction"),
+	MAGERANGE("magerange"),
+	ACID("acid"),
+	ICE("ice"),
```
Comment (saibottig) [inline runelite-api/src/main/java/net/runelite/api/Skill.java]: This does not belong here but should be in Vorkath class enum

**Comment by MagicfTail** [comment]
[Rejected](https://github.com/runelite/runelite/wiki/Rejected-or-Rolled-Back-Features)

**Comment by deathbeam** [comment]
We decided we dont want Vorkath plugin on the original issue, sorry

**Comment by 1337umad** [comment]
runeliteplus accepts rejected features ;)
https://github.com/runelite-extended/runelite/
https://runelitepl.us

**Pull request closed without merging**